taskcluster/ci/config.yml

```diff
@@ -75,6 +75,12 @@ taskgraph:
             default-repository: https://github.com/mozilla-extensions/china-newtab
             default-ref: master
             type: git
+        tabtweak:
+            name: "MozillaOnline Tab Tweak"
+            project-regex: tabtweak$
+            default-repository: https://github.com/mozilla-extensions/tabtweak
+            default-ref: master
```

Comment:
github shows "develop" as the default branch for this repo (with "master" 3 years behind), is this intended?

Reply:
Yes, it's intended. We've always been using `develop` & `master` branches in this way, where the `master` branch is seen as a release branch, and only updated when a new version is ready.